src/coreclr/pal/src/exception/remote-unwind.cpp

```diff
@@ -2533,12 +2290,6 @@ PAL_VirtualUnwindOutOfProc(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *cont
         goto exit;
     }
 
-    UnwindContextToContext(&cursor, context);
```

Comment:
Why did you delete this? It doesn't make sense to unwind and then not update the context.

Reply:
Ah, it seemed unused but looking around, it appears UnwindThread uses it.